katcp/server.py

```diff
@@ -34,9 +34,12 @@
 from .sampling import format_inform_v5, format_inform_v4
 from .core import (SEC_TO_MS_FAC, MS_TO_SEC_FAC, SEC_TS_KATCP_MAJOR,
                    VERSION_CONNECT_KATCP_MAJOR, DEFAULT_KATCP_MAJOR)
-from .version import VERSION, VERSION_STR
 from .kattypes import (request, return_reply)
 
+# 'import katcp' so that we can use katcp.__version__ later
+# we cannot do this: 'from . import __version__' because __version__
+# does not exist at this stage
+import katcp
```

Comment:
I am not sure why this is needed or why __version__ does not exist at this point in time, but I cannot import __version__ in server.py from katcp because it does not exist yet. Is this good enough, or do you know of a better way?